ledger/ledger-api-test-tool/README.md

```diff
@@ -23,93 +18,28 @@ It wraps
 into a standalone command line tool with embedded
 `//ledger/ledger-api-integration-tests:SemanticTests.dar`.
 
-## Usage
-
-*To be moved to SDK DAML documentation.*
-
-### Downloading the tool
-
-TODO
-
-### Extracting `.dar` file required to run the tests
-
-Before you can run the Ledger API test tool on your ledger, you need to load a
-specific set of DAML templates onto your ledger.
-
-1. Obtain the corresponding `.dar` file with:
-
-        ledger-api-test-tool -x
-
-2. Load the file `SemanticTests.dar` created in the current directory into your
-   Ledger.
-
-### Running the tool against a custom Ledger API endpoint
-
-Run this command to test your Ledger API endpoint exposed at host `<host>` and
-at a port `<port>`:
-
-    ledger-api-test-tool -h <host> -p <port>
+### Running standalone
```

Comment:
I'd suggest to reference docs here as well. This way people update the docs that users see instead of internal docs if something change wrt how to run the tool.

Reply:
In the rendered view the docs are linked just few lines above.